### ISSUE TITLE
Update to umano/AndroidSlidingUpPanel v3.3.1

### ISSUE THREAD

**Issue by aantoine** [TR4Android/AndroidSlidingUpPanel#26]
Will this library be updated to the Umano v.3.3.1??

**Comment by taltstidl** [comment]
Yes, it will. Currently I'm on vacation with no computer, so I can't give you an estimated time.

**Comment by taltstidl** [comment]
@aantoine @felipinbombin Just a heads up, I'm delaying the update until the issue with the NineOldAndroids removal is solved in the original library (see https://github.com/umano/AndroidSlidingUpPanel/commit/384fefd4510224d4d2b6e9baf36bbb4e112962d8).
